jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/bigquery/BigQueryQueryRewriterTest.java

```diff
@@ -0,0 +1,121 @@
+package com.exasol.adapter.dialects.bigquery;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+import java.sql.*;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.*;
+import org.junit.jupiter.params.*;
+import org.junit.jupiter.params.provider.*;
+import org.mockito.*;
+import org.mockito.junit.jupiter.*;
+import org.mockito.quality.*;
+
+import com.exasol.*;
+import com.exasol.adapter.*;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.jdbc.*;
+import com.exasol.adapter.sql.*;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+class BigQueryQueryRewriterTest extends AbstractQueryRewriterTest {
+    private QueryRewriter queryRewriter;
+    @Mock
+    private ResultSet mockResultSet;
+    @Mock
+    private ResultSetMetaData mockResultSetMetaData;
+    @Mock
+    private Statement mockStatement;
```

Comment:
We have a `statement` from the `AbstractQueryRewriterTest` and a `mockStatement` in this class. We need to consolidate this.